### ISSUE TITLE
Multi-Select for Requests

### ISSUE THREAD

**Issue by Joly0** [Fallenbagel/jellyseerr#1032]
### Description

Hey, i just implemented SuggestArr https://github.com/giuseppe99barchetta/SuggestArr which scans your library and uses the tmdb api to add new requests based on your available movies to jellyseer. Problem is, it adds quite a few movies and tv shows in one run. There are often alot of movies or tv shows that i have to manually delete.

### Desired Behavior

Would be great to have an option to select multiple requests and remove the request or remove them from radarr/sonarr all at once

### Code of Conduct

- [X] I agree to follow Jellyseerr's Code of Conduct

**Comment by gauthier-th** [comment]
This is probably more a SuggestArr related issue than a Jellyseerr one?

**Comment by Joly0** [comment]
This is correct and i already reported the issue to suggestarr, but it would still be nice to mass approve/deny requests in jellyseer
